autogpt_platform/backend/backend/util/json.py

```diff
@@ -95,3 +96,19 @@ def convert_pydantic_to_json(output_data: Any) -> Any:
     if is_list_of_basemodels(output_data):
         return [item.model_dump() for item in output_data]
     return output_data
+
+
+def SafeJson(data: Any) -> Json:
+    """Safely serialize data and return Prisma's Json type."""
+    if isinstance(data, BaseModel):
+        return Json(
+            data.model_dump(
+                mode="json",
+                warnings="error",
+                exclude_none=True,
+                fallback=lambda v: None,
+            )
+        )
+    # Round-trip through JSON to ensure proper serialization with fallback for non-serializable values
```

Comment:
If we are doing things like this, we may want to use orjson lib as its way faster then default python one:

https://github.com/ijl/orjson?tab=readme-ov-file#performance